src/org/labkey/test/components/ui/grids/GridBar.java

```diff
@@ -205,7 +205,10 @@ public void doMenuAction(String buttonText, List<String> menuActions)
 
         if(found)
         {
-            multiMenu.doMenuAction(menuActions);
+            if (menuActions.size() == 1)
+                multiMenu.doMenuAction(menuActions.get(0));
+            else
+                multiMenu.doMenuAction(menuActions.get(0), menuActions.get(1));
```

Comment:
Make sure we didn't miss some place that does pass in three actions
```suggestion
            if (menuActions.size() == 1)
                multiMenu.doMenuAction(menuActions.get(0));
            else if (menuActions.size() == 2)
                multiMenu.doMenuAction(menuActions.get(0), menuActions.get(1));
            else
                throw new IllegalArgumentException();
```

Reply:
Yes, I kind of went back and forth over calling dropdown-section-toggles menuItems and toggles, wanted to avoid conflating the menu-item-toggles with the menu toggle
Also some of those in there were copy-pasta problems my eyeballs just glazed over.

Let's call dropdown-section-toggles toggles now